internal/daemon/daemon_test.go

```diff
@@ -290,7 +290,7 @@ func TestQuit(t *testing.T) {
 			// drop connection
 			disconnectClient()
 
-			require.Eventually(t, serverHasQuit, 100*time.Millisecond, 10*time.Millisecond, "Server should quit with no more active connection")
+			require.Eventually(t, serverHasQuit, 1*time.Second, 10*time.Millisecond, "Server should quit with no more active connection")
```

Comment:
Make sure you use `testutils.MultipliedSleepDuration` for all these values (and others around)

Reply:
Eh, I would prefer not fiddling with these fragile timeouts. I didn't add anything here (and I don't like tests which only work because of timeouts), I only bumped it because I've seen it fail in CI.

Reply:
I totally agree, and my fight over sleeps never ending, but still since we change those it's a good thing to make them adjust depending on the environment.

If they fail, then they're broken anyways